src/swagger-specs/destinations.yaml

```diff
@@ -34,81 +34,313 @@ servers:
+                $ref: '#/components/examples/Patch_Connection_Replace_Auth_Params'
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Patch_Connection_Response'
+              examples:
+                Patch_Connection_Response:
+                  $ref: '#/components/examples/Patch_Connection_Response'
+        '401':
+          description: The token provided in the Authorization header is invalid.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              example:
+                error_code: 401013
+                message: "Oauth token is not valid"
+        '403':
+          description: You are not authorized to perform this operation 
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '404':
+          description: The specified base connection ID was not found. The resource may have been deleted, or the requested path was entered incorrectly.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              example:
+                errorMessage: "connections not found with id = [ca5c6e9a-8559-4a08-97a5-f15b756da600-]"
+                errorDetails: "connections not found with id = [ca5c6e9a-8559-4a08-97a5-f15b756da600-]"
+        '412':
+          description: >
+            Pre-condition failed error. This can occur if the supplied eTag does not match the latest eTag for the resource. 
+            To resolve this, you need to fetch the entity again, make sure that your changes are compatible with new version of the entity, then place the new etag in the If-Match header, and finally make the API call.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              example:
+                errorMessage: Supplied version=[\"04001277-0000-0200-0000-62d9b13e00001\"] does not match the current version on entity=[\"0400117f-0000-0200-0000-62d9bca50000\"]
+                errorDetails: Supplied version=[\"04001277-0000-0200-0000-62d9b13e00001\"] does not match the current version on entity=[\"0400117f-0000-0200-0000-62d9bca50000\"]
+                
+        '500':
+          description: An Internal error occurred.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+      # externalDocs:
+      #   description: Learn more about user operations provided by this API.
+      #   url: http://api.example.com/docs/user-operations/
+
+    post:
+      tags:
+        - Base connections
+      summary: Create a new target base connection
```

Comment:
For creation/POST calls, typically we don't need to include "new": "Create a target base connection"

Also noticed that the listed order of operations for Base Connections is off from what we normally do (the other sections are fine). Ideally it should be:

- GET (listing)
- GET (lookup)
- POST
- PUT
- PATCH
- DELETE